lerna-semver-sync.js

```diff
@@ -86,15 +86,20 @@ function getCommonRange (compoundRanges) {
     }, {});
 
     forEach(majorVersions, (ranges, majorVersion) => {
-        majorVersions[majorVersion] = intersect(...ranges);
+        try {
+            majorVersions[majorVersion] = intersect(...ranges);
```

Comment:
So this is really the main part of it - we don't want to throw and halt.  We want to let the process continue and then let the built in logging take place.

The comment you put in the catch block looks good - but you can remove the passing of `name`

Reply:
updated it, thanks!